0-module-and-module-source.emu

```diff
@@ -537,27 +578,29 @@ location: https://tc39.es/proposal-compartments/
         </ul>
 
         <emu-clause id="sec-module">
-            <h1>Module ( _moduleSource_, _referral_, _importHook_ [, _importMeta_ ] )</h1>
-            <p>When the `Module` function is called with the arguments _moduleSource_, _referral_, _importHook_ and _importMeta_, the following steps are taken:</p>
+            <h1>Module ( _moduleSource_, _handler_ )</h1>
```

Comment:
where did you put the referral?

Reply:
You can store it on `handler`, and access it in hooks as `this.referral`